CHANGELOG.md

```diff
@@ -1 +0,0 @@
-docs/changelog.md
```

Comment:
How does the symbolic link for the change log work?

Reply:
I think you have to keep the file as it is and make your changes in `docs/changelog.md`

@joelhawksley could it be the other way around (put the symlink in the docs folder and keep the real changelog at the root)? It's confusing when you want to access the changelog from GitHub web UI.

Reply:
@johannesengl we need to update the file in `docs`, as @Spone mentioned. 

@Spone I tried doing the symlink in the other direction, but Jekyll was unhappy with that setup due to Github Pages running Jekyll in `safe` mode.